### DISCUSSION TITLE
Applicazione ritenuta d'acconto del 4% tramite API

### DISCUSSION THREAD

**Discussion by rastellid** [fattureincloud/api#233] [Q&A] [answered]
Ciao 

Tramite le API v2 quando creo una fattura, ho la necessita di applicare una ritenuta d'acconto del 4%. 

Quali chiavi devo impostare correttamente nella body request ?

Grazie anticipatamente

**Comment by emajo** [comment, marked as answer]
Ciao @rastellid per applicare una ritenuta d'acconto sono due i campi necessari:
`withholding_tax`: la percentuale della ritenuta, in questo caso 4
`withholding_tax_taxable`: la percentuale dell'imponibile, in questo caso 100

Grazie,
Emanuele.

**Reply by rastellid** [reply]
Grazie per la celerità e la risposta.

Dovrei anche cambiare al causale passando dal prefisso ZO al W a qualche chiave faccio rifermento ?

Noto che c'è un `ei_withholding_tax_causal:string|null`

Un formato corretto potrebbe essere `"ei_withholding_tax_causal":"W"` ?

**Comment by rastellid** [comment]
Grazie per la celerità e la risposta.

Sempre per la ritenuta d'acconto, dovrei anche cambiare al causale passando dal prefisso ZO al W a qualche chiave faccio rifermento ?

Noto che c'è un ei_withholding_tax_causal:string|null

Un formato corretto potrebbe essere "ei_withholding_tax_causal":"W" ?

**Reply by emajo** [reply]
Esattamente, `"ei_withholding_tax_causal":"W"` è corretto.
